Tests/LibWasm/test-wasm.cpp

```diff
@@ -62,3 +95,102 @@ TESTJS_GLOBAL_FUNCTION(compare_typed_arrays, compareTypedArrays)
     auto& rhs_array = static_cast<JS::TypedArrayBase&>(*rhs);
     return JS::Value(lhs_array.viewed_array_buffer()->buffer() == rhs_array.viewed_array_buffer()->buffer());
 }
+
+void WebAssemblyModule::initialize(JS::GlobalObject& global_object)
+{
+    Base::initialize(global_object);
+    define_native_function("getExport", get_export);
+    define_native_function("invoke", wasm_invoke);
+}
+
+JS_DEFINE_NATIVE_FUNCTION(WebAssemblyModule::get_export)
+{
+    auto name = vm.argument(0).to_string(global_object);
+    auto this_value = vm.this_value(global_object);
+    auto instance = static_cast<WebAssemblyModule*>(this_value.to_object(global_object));
+    if (!instance) {
+        vm.throw_exception<JS::TypeError>(global_object, "Not a WebAssemblyModule");
+        return {};
+    }
+    for (auto& entry : instance->m_machine.module_instance().exports()) {
+        if (entry.name() == name) {
+            auto& value = entry.value();
+            if (auto ptr = value.get_pointer<Wasm::FunctionAddress>())
+                return JS::Value(static_cast<unsigned long>(ptr->value()));
+            vm.throw_exception<JS::TypeError>(global_object, String::formatted("'{}' does not refer to a function", name));
+            return {};
+        }
+    }
+    vm.throw_exception<JS::TypeError>(global_object, String::formatted("'{}' could not be found", name));
+    return {};
+}
+
+JS_DEFINE_NATIVE_FUNCTION(WebAssemblyModule::wasm_invoke)
+{
+    auto address = static_cast<unsigned long>(vm.argument(0).to_number(global_object).to_double(global_object));
```

Comment:
`to_number()` can blow up in your face, needs exception check before `to_double()`'ing.

Reply:
hmm...can I just call `.to_double()` on the value instead?

Reply:
yep, good idea! still needs an exception check, but you'll get a double directly (0 if error)